samples/fbo_new_blend_equation.html

```diff
@@ -0,0 +1,253 @@
+            0.0, 1.0,
+            1.0, 1.0,
+            1.0, 0.0,
+            1.0, 0.0,
+            0.0, 0.0,
+            0.0, 1.0
+        ]);
+        gl.bufferData(gl.ARRAY_BUFFER, texcoords, gl.STATIC_DRAW);
+        gl.bindBuffer(gl.ARRAY_BUFFER, null);
+
+        // -- Initilize vertex array
+
+        var vertexArray = gl.createVertexArray();
+        gl.bindVertexArray(vertexArray);
+
+        var vertexPosLocation = 0; // set with GLSL layout qualifier
+        gl.bindBuffer(gl.ARRAY_BUFFER, vertexPosBuffer);
+        gl.vertexAttribPointer(vertexPosLocation, 2, gl.FLOAT, false, 0, 0);
+        gl.enableVertexAttribArray(vertexPosLocation);
+        gl.bindBuffer(gl.ARRAY_BUFFER, null);
+
+        var vertexTexLocation = 4; // set with GLSL layout qualifier
+        gl.bindBuffer(gl.ARRAY_BUFFER, vertexTexBuffer);
+        gl.vertexAttribPointer(vertexTexLocation, 2, gl.FLOAT, false, 0, 0);
+        gl.enableVertexAttribArray(vertexTexLocation);
+        gl.bindBuffer(gl.ARRAY_BUFFER, null);
+
+        gl.bindVertexArray(null);
+
+        // -- Load texture then render
+        var imageUrl = '../assets/img/Di-3d.png';
+        var texture;
+        loadImage(imageUrl, function(image) {
+            texture = gl.createTexture();
+            gl.bindTexture(gl.TEXTURE_2D, texture);
+            gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MAG_FILTER, gl.LINEAR);
+            gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MIN_FILTER, gl.LINEAR);
+            gl.texImage2D(
+                gl.TEXTURE_2D,
+                0, // Level of details
+                gl.RGBA, // Format
+                gl.RGBA,
+                gl.UNSIGNED_BYTE, // Size of each channel
+                image
+            );
+
+            gl.generateMipmap(gl.TEXTURE_2D);
+            render();
+        });
+
+        function render() {
+            // Clear color buffer
+            gl.clearColor(0.5, 0.0, 0.0, 1.0);
+            gl.clear(gl.COLOR_BUFFER_BIT);
+
+            // Bind program
+            gl.useProgram(program);
+
+            var matrix = new Float32Array([
+                1.0, 0.0, 0.0, 0.0,
+                0.0, 1.0, 0.0, 0.0,
+                0.0, 0.0, 1.0, 0.0,
+                0.0, 0.0, 0.0, 1.0
+            ]);
+            gl.uniformMatrix4fv(uniformMvpLocation, false, matrix);
+            gl.uniform1i(uniformDiffuseLocation, 0);
+
+            gl.activeTexture(gl.TEXTURE0);
+            gl.bindTexture(gl.TEXTURE_2D, texture);
+
+            gl.bindVertexArray(vertexArray);
+
+            gl.enable(gl.BLEND);
+            
+            for (var i = 0; i < Corners.MAX; ++i) {
+                gl.viewport(viewport[i].x, viewport[i].y, viewport[i].z, viewport[i].w);
+                
+                if(i == Corners.TOP_LEFT)
+                {
+                    //pass
+                }
+                else if(i == Corners.TOP_RIGHT)
+                {
+                    gl.drawArraysInstanced(gl.TRIANGLES, 0, 6, 1);
```

Comment:
This is simply because Christophe used the draw calls like this in his code. 
Yeah you are right. I don't think this draw instanced call makes sense here. 
I will fix it and take care of the merging